cvat/apps/webhooks/signals.py

```diff
@@ -4,17 +4,24 @@
 
 import hashlib
 import hmac
-from http import HTTPStatus
 import json
+from copy import deepcopy
+from http import HTTPStatus
 
 import django_rq
 import requests
-from django.dispatch import Signal, receiver
 from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from django.db.models.signals import (post_delete, post_save, pre_delete,
+                                      pre_save)
+from django.dispatch import Signal, receiver
 
-from cvat.apps.engine.models import Project
+from cvat.apps.engine.models import Comment, Issue, Job, Label, Project, Task
 from cvat.apps.engine.serializers import BasicUserSerializer
-from cvat.apps.organizations.models import Organization
+from cvat.apps.events.handlers import (_get_current_request, _get_serializer,
```

Comment:
If these are used in another module, then their names shouldn't begin with an underscore.